modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro

```diff
@@ -258,6 +258,34 @@ definition {
 		MenuItem.clickNoError(menuItem = "Duplicate");
 	}
 
+	macro editFilterViaUI {
+		ObjectCustomViews.assertVisibleEntriesOnFilters(filterBy = "${filterName}");
+
+		Click(
+			key_cardName = "Filters",
+			locator1 = "ObjectCustomViews#SELECT_DROPDOWN_ON_LABEL");
+
+		DropdownMenuItem.click(menuItem = "Edit");
+
+		if (isSet(filterValue)) {
+			Click(
+				key_filterLabel = "Value",
+				locator1 = "ObjectCustomViews#SELECT_FILTER_GENERIC");
+
+			ObjectCustomViews.assertDefaultMetadataOnFilterValue();
+
+			for (var filterValueList : list "${filterValue}") {
+				Click.clickAtNotVisible(
+					key_columnOption = "${filterValueList}",
+					locator1 = "ObjectCustomViews#CHECKBOX_COLUMN_OPTION");
+			}
+		}
+
+		Click(
+			key_name = "Save",
+			locator1 = "Button#GENERAL_BUTTON_MODAL_FOOTER");
+	}
+
```

Comment:
Do we need to extract this into a macro? The code isn't being re-used and you're using tasks to explain what's happening in the test so I don't think we need to create this macro, unless I'm missing something. 

Out of curiosity, is the expected coding practice in Brazil to not use functions at the test case level?